docs/source/faq/installation.md

```diff
@@ -0,0 +1,172 @@
+# FAQ: Installation
```

Comment:
Maybe for a different PR, but I would like to see a point about configuring IDE's like vscode for working with manim. I've written a blog post recently: https://naveen521kk.github.io/manim/manim-configuration-for-vscode/ either linking to directly or copying contents from it should work. A similar point should be written for Pycharm also.

Reply:
Good idea! Also fits well in the installation FAQ!